db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.2].define(version: 2024_12_16_224825) do
+ActiveRecord::Schema[8.0].define(version: 2025_01_08_111200) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
```

Comment:
It looks like the change in your previous commit here was due to a Rails 8 change (we had no migration since our switch to Rails 8), though both this version and the previous one seem to work just fine.
```suggestion
  enable_extension "pg_catalog.plpgsql"
```